development/workflow.rst

```diff
@@ -0,0 +1,668 @@
+
+Project
+   This should be set to **Data Management**, unless you are creating a ticket for a different LSST subsystem.
+Issue Type
+   If the work is associated with an epic, the issue type is a 'Story.'
+   For emergent work, 'Bug' or 'Enhancement' can be used (see above for semantics).
+Summary
+   This is the ticket's title and should be written to help colleagues browsing JIRA dashboards.
+Components
+   You should choose from the pre-populated list of components to specify what part of the DM system the ticket relates to.
+   If in doubt, ask your T/CAM.
+Assignee
+   Typically you will assign yourself (or your T/CAM will assign you) to a ticket.
+   You can also assign tickets to others.
+   If you are uncertain about who the assignee should be you can allow the ticket to be automatically assigned (which defaults to the component's T/CAM; `RFC-51 <https://jira.lsstcorp.org/browse/RFC-51>`_).
+Description
+   The description should provide a clear description of the deliverable that can serve as a definition of 'Done.'
+   This will prevent scope creep in your implementation and the code review.
+   For stories, you can outline your implementation design in this field.
+   For bug reports, include any information needed to diagnose and reproduce the issue.
+   Feel free to use `Atlassian markup syntax <https://jira.lsstcorp.org/secure/WikiRendererHelpAction.jspa?section=texteffects>`_. 
+Story Points
+   Use this field, at ticket creation time, to **estimate** the amount of effort involved to accomplish the work.
+   Keep in mind how *velocity* (see above) converts story points into real-world days.
+Labels
+   Think of labels as tags that you can use to sort your personal work.
+   Unlike the Component and Epic fields, you are free to create and use labels in any way you see fit.
+Linked Issues
+   You can express relationships between JIRA issues with this field.
+   For example, work that implements an RFC should link to that RFC.
+   You can also express dependencies to other work using a 'is Blocked by' relationship.
+Epic Link
+   If the ticket is a story, you must specify what epic it belongs to with this field.
+   By definition, bug or enhancement-type tickets are not associated with an epic.
+Team
+   You must specify which DM team is doing the work with this field, for accounting purposes.
+   The owner of the epic should be consistent with the team working on a ticket.
+
+.. _workflow-jira-ticket-status:
+
+Ticket status
+-------------
+
+Tickets are created with a status of 'Todo.'
+
+Once a ticket is being actively worked on you can upgrade the ticket's status to 'In Progress.'
+
+.. _git-branching:
+
+DM Git Branching Policy
+=======================
+
+Rather than forking LSST's GitHub repositories, DM developers use a *shared repository model* by cloning repositories in the `lsst <https://github.com/lsst>`_, `lsst-dm <https://github.com/lsst>`_, and `lsst-sqre <https://github.com/lsst>`_ GitHub organizations.
```

Comment:
We generally use the fork & PR model for repos under `lsst-sqre`.  The shared model is only needed for repos built by `lsst_build`.